quilt/README.md

```diff
@@ -0,0 +1,241 @@
+# Running Ray with Quilt
+
+Quilt is a way to deploy and network containers, meaning that Quilt will boot
+machines and set up containers on supported cloud providers (AWS, GCE, Azure)
+for you. Learn more about this project [here](http://quilt.io).
+
+## Introduction
+Quilt uses a domain specific language, Stitch (files that end with `.spec`, to 
+specify distributed applications, independent of the specific infrastructure
+they run on. This folder contains [`ray.spec`](ray.spec) and 
+[`ray-runner.spec`](ray-runner.spec). `ray.spec` provides abstractions for
+creating ray containers - the function `ray.New` creates `nWorker` worker nodes,
+and one head node, all prefixed with `prefix`.
+
+`ray-runner.spec` defines the infrastructure When you run `ray-runner.spec`, Quilt will
+
+## Step 1: Installing Quilt
+
+### Install Go
+Quilt supports Go version 1.5 or later.
+
+Install Go using your package manager or on the 
+[Golang website](https://golang.org/doc/install).
+
+#### Set up GOPATH
+We recommend reading the overview to Go workplaces 
+[here](https://golang.org/doc/code.html).
```

Comment:
I'm not sure that people need to understand Go workplaces as Quilt users. Or rather, we'd like to get things so that they don't need to understand.